src/amo/actions/ratingActions.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
+import { SET_USER_RATING } from 'amo/constants';
+
+export const setUserRating = ({ addonID, userRating }) => ({
```

Comment:
This file could probably be named `ratings.js` (it's plural in the other places) since it's in the `actions` folder.